source/Xunit.Gherkin.Quick/Feature.cs

```diff
@@ -12,6 +12,19 @@ namespace Xunit.Gherkin.Quick
 {
     public abstract class Feature
     {
+        /// <summary>Allows you to log extra data to the result of the test.</summary>
+        protected ITestOutputHelper Output { get; set; }
```

Comment:
No need for the setter. Only get is enough since you set it in the constructor. Plus, will be cleaner with minimalist design.